framework/UI/FitnessView.py

```diff
@@ -95,34 +95,38 @@ def Reinitialize(self, parent=None, amsc=None, title=None):
     layout.addWidget(self.gView)
     self.updateScene()
 
-  def saveImage(self):
+  def saveImage(self, filename=None):
     """ Saves the current display of this view to a static image by loading a
```

Comment:
Ah, yes. This will apply to each instance of this function. I will adjust the docstrings here and in the other classes where this exists.